topics/raw.md

```diff
@@ -70,9 +70,9 @@ a variable definition based on the class of event.
 
 
 <br/>
-<a name="lti-link-followed"></a>
+<a name="lti-links"></a>
 
-### Events with class `lti.link.followed`
+### Events with class `lti.link.followed`, `lti.link.best_guess.found_list`, `lti.link.best_guess.search`, `lti.link.best_guess.homepage`
```

Comment:
As well as being less verbose, this also means that a data analyst will become aware that there are multiple similar events that they might want to think about including in their analysis.